### PR TITLE
Scaling on rotation blog post

### PR DESCRIPTION
**Pull request by lakesare**

### CODE REVIEW

File: blog/2023-10-25-image-editor.md [hunk 1]
```diff
@@ -125,16 +137,15 @@ corners disappear.
...
+`scale(Math.max(scalingFactor1, scalingFactor2))`. These two scaling factors are
+calculated very similarly, so we’ll only focus on calculating only one of them
+in this tutorial (although the full solution is given in the conclusion).
 
 In the images below, the <span style={{ color: `rgb(127, 194, 65)` }}>green
 rectangle</span> represents the desired dimensions of our image after it’s
-scaled. Scaling operation is defined in such a way that our scaling factor is
-`H/h`. We already know `h` (it’s the height of our image!), so we want to find
-`H`.
+scaled. Luckily for us, the scaling function is defined in such a way that our
```
Comment (lakesare) [inline blog/2023-10-25-image-editor.md:146]: Came up with a better alternative than both of our versions, tell me what you think:

Our scaling function (and, hopefully, yours - in general that's how scaling is defined in linear algebra) is defined in such a way that if we have the image of height `h`, and we want to scale it up to height `H`, we want to execute `.scale(H/h)`. We already know `h`, the height of our image, and now we want to find `H` - all of the further steps in this tutorial will be automatic - we know all the angles in this image, we know the image's width and height, and we want to find `H`.

File: blog/2023-10-25-image-editor.md [hunk 2]
```diff
@@ -177,7 +188,7 @@ for <code>90 - α</code>:
   </tbody>
 </table>
 
-and then find our `H` by adding two large-triangle sides:
+We can then find `H`, by adding the sides of the two outer triangles.
```
Comment (lakesare) [inline blog/2023-10-25-image-editor.md:191]: Maybe:

We can then find `H`, by adding two outer sides of these triangles.

File: blog/2023-10-25-image-editor.md [hunk 3]
```diff
@@ -197,9 +208,9 @@ and then find our `H` by adding two large-triangle sides:
   </tbody>
 </table>
 
-So, now we have our desired `H`! One of our scaling factors is `H/h`. Another
-scaling factor of ours will be `W/w`, calculated similarly.  
-The full solution will look as following:
+So, now we have our desired `H`! One of our scaling factors is `H/h`. Our other
+scaling factor is `W/w`, which follows a similar process. Below is a rough
+solution.
```
Comment (lakesare) [inline blog/2023-10-25-image-editor.md:213]: It makes it sound like "below is a calculation of the `W/w` part", while it's not, it's just a full solution. Also - it's not a rough solution, it's a proper solution.

File: blog/2023-10-25-image-editor.md [hunk 4]
```diff
@@ -232,9 +243,9 @@ cropper.scale(scaleFactor);
 ```
 
 You can see the full version
-[on github](https://github.com/transloadit/uppy/blob/12e08ada02b9080bd5e1d19526bdf8a2010e62a1/packages/%40uppy/image-editor/src/utils/getScaleFactorThatRemovesDarkCorners.js).
+[on GitHub](https://github.com/transloadit/uppy/blob/12e08ada02b9080bd5e1d19526bdf8a2010e62a1/packages/%40uppy/image-editor/src/utils/getScaleFactorThatRemovesDarkCorners.js).
 
 <details>
-  <summary>Bonus content: our founder’s (Tim Koschuetzki) scribble notes with the solution</summary>
+  <summary>Bonus content: our founder’s (Tim Koschuetzki) inital scribbled notes with the solution</summary>
```
Comment (lakesare) [inline blog/2023-10-25-image-editor.md:249]: "init**i**al" typo

**Review by lakesare — COMMENTED** [review]
Would you look over these please & incorporate them as you see fit?

File: blog/2023-10-25-image-editor.md [hunk 5]
```diff
@@ -50,57 +50,69 @@ Below you see how Uppy’s image editor handled image rotations prior to our
   </tbody>
 </table>
 
-This UI is present in many image editors - for example, the default image
-editors on ios and android both employ it. We implemented this in Uppy’s image
-editor last week, and the solution turned out non-trivial. Bearing in mind this
-is a pretty ubiquitous task to solve for image editors, we decided to write out
-a post about it instead of leaving the solution in internal notes.
+Lets dive into some of the finer technical details, so you can follow along and
```
[the remark this answers is not in the document]
Comment (Missing-Tech) [reply blog/2023-10-25-image-editor.md:53]: Good spot 😄

File: blog/2023-10-25-image-editor.md [hunk 6]
```diff
@@ -50,57 +50,69 @@ Below you see how Uppy’s image editor handled image rotations prior to our
...
+implement this feature into your own image editor.
+
+<!--truncate-->
```
[the remark this answers is not in the document]
Comment (Missing-Tech) [reply blog/2023-10-25-image-editor.md:56]: I believe this cuts off the blurb on the blogs page. e.g. here is the blog page with/without truncate:
image
image

File: blog/2023-10-25-image-editor.md (hunk 1 above)
Comment (Missing-Tech) [reply blog/2023-10-25-image-editor.md:146]: I like this a lot :) Just added it and changed a word ere and there so it flows a little nicer but definitely a big improvement.

File: blog/2023-10-25-image-editor.md [hunk 7]
```diff
@@ -12,12 +12,12 @@ toc_max_heading_level: 3
 
 ## Introduction
 
-This is a tutorial blog post, I’ll be explaining how we implemented “image
-scaling on rotation” in Uppy, and how you can implement it in your own image
-editor.
+We recently added “image scaling on rotation” to Uppy's
+[Image Editor](https://uppy.io/docs/image-editor/), which means images will
+maintain the full frame, even when rotated.
```
[the remark this answers is not in the document]
Comment (Missing-Tech) [reply blog/2023-10-25-image-editor.md:17]: Took a swing at changing the feel of the introduction a little, would love to know what you think! Also agreed on the phrasing not being so clear 😄

**Comment by Missing-Tech** [comment]
Just pushed another iteration, addressing most of your comments :) Let me know if there's anything you think needs changing and I'd be happy to take another swing at this until we get it perfect

File: blog/2023-10-25-image-editor.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+authors: [evgenia, tim]
+image: '/img/blog/3.4-3.13/single-file-mode.jpg'
+slug: '2023-10-25-image-editor'
+published: true
+toc_max_heading_level: 3
+---
+
+<!--retext-simplify disable prior-to all-of employ very represents appropriate-->
+
+We recently released an “image scaling on rotation” feature, for Uppy's
```
Comment (lakesare) [inline blog/2023-10-25-image-editor.md:13]: I don't think there should be a comma between "feature, for"

**Comment by lakesare** [comment]
@Missing-Tech, would you be able to take it from here, wrap up the blog post & fixed the linters & get it merged?

**Comment by Missing-Tech** [comment]
Sure I'd love to!

**Comment by Missing-Tech** [comment]
Sorry for the delay here! Just fixed some linting errors and it should be merged soon :)

**Comment by Missing-Tech** [comment]
@arturi Is this ok to be merged?

**Pull request merged**